Library/Homebrew/cleanup.rb

```diff
@@ -209,6 +209,19 @@ def cleanup_logs
       end
     end
 
+    def cleanup_unreferenced_downloads
+      return if dry_run?
+      return unless (cache/"downloads").directory?
+
+      downloads = (cache/"downloads").children.reject { |path| path.incomplete? } # rubocop:disable Style/SymbolProc
```

Comment:
Why can't this use `:incomplete?`

Reply:
Because the refinement scope doesn't work like that, sadly. It would call `.incomplete?` in the file where `reject` is defined rather than here.

Reply:
Gotcha. Mind adding a comment above this pointing this out so people don't think it's a smell in future? Thanks!